static/app/components/group/groupSummary.tsx

```diff
@@ -50,6 +50,29 @@ export const makeGroupSummaryQueryKey = (
   },
 ];
 
+/**
+ * Gets the data for group summary if it exists but doesn't fetch it.
+ */
+export function useGroupSummaryData(
+  group: Group,
+  event: Event | null | undefined,
+  forceEvent = false
+) {
+  const organization = useOrganization();
+  const queryKey = makeGroupSummaryQueryKey(
+    organization.slug,
+    group.id,
+    forceEvent ? event?.id : undefined
+  );
+
+  const {data, isPending} = useApiQuery<GroupSummaryData>(queryKey, {
+    staleTime: Infinity,
+    enabled: false,
```

Comment:
I _think_ this should stop it. Query can still run if `refetch` is used anywhere, but I think we're good?

Reply:
just to check my understanding - this is being used to get the group summary if it exists already, but won't create a new summary if one doesn't exist (so we don't need any checks around it)

Reply:
yep, that is the intent and my understanding of how this hook works